src/couchdb/js-design-document.ts

```diff
@@ -1,3 +1,8 @@
+// Functions accessible within CouchDb design documents:
+declare global {
+  function emit(key: unknown, value?: unknown): void
```

Comment:
`key` should be optional to allow `emit()`

Reply:
Okeydokey, done!